src/high-level/high-level.lisp

```diff
@@ -5,6 +5,10 @@
 
 (in-package #:magicl)
 
+(defconstant +double-float-epsilon+ #+sbcl sb-c::double-float-epsilon #-sbcl 1.1102230246251568d-16)
+
+(defconstant +single-float-epsilon+ #+sbcl sb-c::single-float-epsilon #-sbcl 5.960465e-8)
+
```

Comment:
Some reason not to use `cl:{single,double}-float-epsilon` for these?

Reply:
oh, I see now that this was just moved from the high-level tests. carry on.

Reply:
I actually hadn't realized that the epsilons were part of the standard. Thanks for bringing that to my attention.